packages/frontend/src/utils/moonpay.js

```diff
@@ -40,7 +40,12 @@ export const isMoonpayAvailable = async () => {
 };
 
 export const getSignedUrl = async (accountId, redirectUrl) => {
-    const widgetUrl = `${MOONPAY_BUY_URL_PREFIX}&walletAddress=${encodeURIComponent(accountId)}&currencyCode=NEAR&redirectURL=${encodeURIComponent(redirectUrl)}`;
+    const widgetUrl = `${MOONPAY_BUY_URL_PREFIX}`
+        + `&walletAddress=${encodeURIComponent(accountId)}`
+        + `&currencyCode=NEAR`
+        + `&baseCurrencyCode=usd`
```

Comment:
Did we have to add `baseCurrencyCode` here for some reason? What is the impact of this additional URL param?

Reply:
I added `baseCurrencyCode` here to make sure that Moonpay shows the amount in USD since that's what we show on our buttons in the wallet

Reply:
ah -- gotcha :+1: